docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.7'
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+      MYSQL_USER: registrations_db
+      MYSQL_PASSWORD: registrations_db
+      MYSQL_DATABASE: agilebrazil_development
+    volumes:
+      - mysql-data:/var/lib/mysql
+  app:
+    build:
+      context: .
+    command: bundle exec rails s -b 0.0.0.0
+    ports:
+      - "3000:3000"
+    volumes:
+      - .:/app/
+    environment:
+      MYSQL_ROOT_PASSWORD: root
```

Comment:
What is this for? What in the app uses the root and that docker database?

Reply:
Good catch, actually I was using this at the begging and I stopped to use it after creating the `database.docker` file. 
Removing.